sql/moz-fx-data-shared-prod/accounts_backend_external/nonprod_emails_v1/metadata.yaml

```
@@ -0,0 +1,22 @@
+---
+friendly_name: Emails table from FxA database in stage environment
+description: >
```

Need to respect newlines for the list items to not get condensed into the first paragraph.
```suggestion
description: |-
```